spec/takeaway_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:menu) { double :menu, content: {"Fish and chips" => 7, "Steak and chips" => 10,
+                                       "Shepherd's Pie" => 5, "Bangers and mash" => 5} }
```

Comment:
Line is too long. [88/80]<br>Expression at 6, 88 should be on its own line.<br>Space inside } missing.